### ISSUE TITLE
BUG: alma PI queries are broken

### ISSUE THREAD

**Issue by bsipocz** [astropy/astroquery#2584]
The test querying based on the PI name is broken, and I couldn't trivially figure out the column name we should use instead.

To reproduce run:

`pytest astroquery/alma/tests/test_alma_remote.py::TestAlma::test_misc --remote-data`

or the examples from the docs in the section https://astroquery.readthedocs.io/en/latest/alma/alma.html#querying-by-other-parameters.

The test send the tap query: `"select * from ivoa.obscore WHERE obs_creator_name LIKE '%Bally%' AND science_observation='T' AND data_rights='Proprietary'"`, which also results an empty table

cc @andamian @keflavich

**Comment by keflavich** [comment]
This is likely a service failure; ALMA was hit by a "cyberattack" a few weeks ago and services have yet to be restored.  Let's wait to take action on this until their servers are back to 100%

**Comment by bsipocz** [comment]
it's still a 200 response, so are you sure it's a service outage? (If it's, it's a different upstream bug, I mean it should return an error for service outage rather than a valid response)

**Comment by keflavich** [comment]
Ah, that's odd.  No, I'm not sure.  It's possible, though, that the archive is just returning blank results now where it shouldn't be....

**Comment by keflavich** [comment]
https://almascience.nrao.edu/news/alma-services-affected-by-cyberattack btw

**Comment by bsipocz** [comment]
I see this empty table in logs from early Oct, so it predates the attack.

**Comment by keflavich** [comment]
Ah, I didn't realize that.  OK, something is wrong.  Thanks for noticing.

**Comment by at88mph** [comment]
I can no longer reproduce this.  It may have been a transient issue with their services.  My tests run in the current state.  Should we leave this open in case it happens again?

**Comment by bsipocz** [comment]
I still get empty results running the examples from the docs. Is this expected to return nothing?
```
>>> from astroquery.alma import Alma
>>> from astropy import units as u
>>> Alma.query_region('W51', radius=25*u.arcmin, pi_name='*Ginsburg*')
<Table length=0>
obs_publisher_did obs_collection facility_name instrument_name ... lastModified collections pi_userid
                                                               ...                                   
      str33            str4           str3           str4      ...    str32        str128     str256 
----------------- -------------- ------------- --------------- ... ------------ ----------- ---------
```

**Comment by andamian** [comment]
I've just noticed one remote-data int test that's failing on this. The service doesn't seem to return the correct PI name anymore. We are looking into it.

**Comment by bsipocz** [comment]
(unrelated, but there are a couple of other failing remote data for alma)

**Comment by andamian** [comment]
Yes, thanks - I've noticed that too. One is related to a column concatenate issue on the server side that we are working on it. I'm still investigating the other one.
